### PR TITLE
Introducing cobra commands for UI.

### PR DESCRIPTION
**Pull request by dbaltas**

### CODE REVIEW

File: cmd/deploy.go [hunk 1]
```diff
@@ -0,0 +1,182 @@
...
+			directory,
+			branchMap,
+			viper.GetString("release.on-deploy.body-branch-suffix-find"),
+			viper.GetString("release.on-deploy.body-branch-suffix-replace"),
+			viper.GetString("github.access-token"),
+		)
+	},
+}
+
+func deployBranches(organization string, remote string, releaseRepo string, baseBranch string, branches []string, releaseOffset string, releaseInterval string, directory string, branchMap map[string]string, suffixFind string, suffixReplace string, githubAccessToken string) {
```
Comment (mantzas) [inline cmd/deploy.go]: should be a struct. way to many arguments. Maybe it does too much and should be broken up into smaller components?

File: readme.md [hunk 2]
```diff
@@ -16,20 +16,34 @@ $ go build -o ergo main.go
...
+On cases where deployment is done by pushing on a git branch:
+* it can draft a github release,
+* deploy on multiple branches and update the release notes with the time of release
+
+Usage:
+  ergo [flags]
+  ergo [command]
+
+Available Commands:
+  deploy      Deploy base branch to target branches
```
Comment (mantzas) [inline readme.md:31]: maybe create a command release that has deploy and draft as sub commands?

File: repo/repo.go [hunk 3]
```diff
@@ -54,7 +54,7 @@ func LoadOrClone(repoURL string, directory string, remoteName string, skipFetch
 	}
 
 	if !skipFetch {
-		fmt.Printf("Fetching remote %s (use -skipFetch to skip)\n", remoteName)
+		fmt.Printf("Fetching remote %s (use --skipFetch to skip)\n", remoteName)
```
Comment (mantzas) [inline repo/repo.go]: suggestion: you can use std lib's log instead of fmt

File: cmd/deploy.go (hunk 1 above)
Comment (dbaltas) [reply cmd/deploy.go]: I 'll wait a bit on that, until I figure out what to do with this func. It doesn't belong inside the cmd package anyway.

File: repo/repo.go (hunk 3 above)
Comment (dbaltas) [reply repo/repo.go]: yeap, let's talk about this. Maybe a more advanced logger can be added.

**Pull request merged**